Update oxygen_pre_deployment.md (#101)
OceanGlidersCommunity/Oxygen_SOP: sections/oxygen_pre_deployment.md

# Pre-deployment operations and calibrations

## Storage and cleaning

> - Optode foils typically drift more while in storage than while in use, the reasons for this are thought to be due to exposure to UV radiation and dry air {cite}`Bittig2018`, {cite}`Aanderaa2018`. 

Reviewer: ```suggestion
Optode foils typically drift more while in storage than while in use, the reasons for this are thought to be due to exposure to UV radiation and dry air {cite}`Bittig2018`, {cite}`Aanderaa2018`. 
```

We recommend that all the available optodes sensors should be stored away from the light (especially fluorescent lights), keep the foil humid and use the plastic caps provided with the sensor. It is highly recommend a two-point calibration prior to each deployment.

> Optode foil sensors should be cleaned before storage and stored with black caps on including some Milli-Q water, or tap water, or with a piece of wet cotton taped against the foil. 

Reviewer: ```suggestion
Sensors should be cleaned before storage and stored with black caps on, including some Milli-Q or tap water or with a piece of wet cotton taped against the foil. 
```

Issues with Winkler method at low oxygen concentrations (below 1 $\mu$mol/kg, some researchers consider that values below 20 $\mu$mol/kg shouldn't be use for the sensor calibration): 
- There are various issues with Winkler at low oxygen concentrations as Winkler always biases towards too high oxygen: the detection limit of the method is around 1 uM {cite}`Langdon2010`, the oxygen absorbed in the plastic of the Niskin bottles might be transferred to  the water sampled (reference needed).

> - For example,within the core of the Peruvian oxygen minimum zones oxygen concentrations at nmol levels are present

Reviewer 2: ```suggestion
- For example, within the core of the Peruvian oxygen minimum zones oxygen concentrations at nmol levels are present
```

other approaches for 0 % in-situ calibration {cite}`Revsbech2009` and {cite}`Thomsen2016`.
- STOX sensor, new low-oxygen sensing foils (0-10% saturation) from Aanderaa.

#### 100 / 0 % saturation protocol

> This method is necessary to check the drift of oxygen measurements over time as the foil wears out. It’s recommended to do it before the deployment and after recovery. NOTE: Sensor foil must be wet during all protocol steps.   

Reviewer: ```suggestion
This method is necessary to check the sensor drift over time as the foil wears out. It’s recommended to do it before the deployment and after recovery. NOTE: Sensor foil must be wet during all procedure steps.   
```

Reviewer: Community uses sensor drift mostly, is this  OK?

Some information to read before we proceed with the calibration:

> 1. For a 100 % bubbled bath, connect an aquarium pump to the tube which at the end you have fitted with a porous stone (bubble dispenser). This will create small air bubbles that are sufficient to equilibrate the water rapidly. 

Reviewer 2: I prefer to not talk to the user in "you" but keep in passive as it is.

Author: You are right @soerenthomsen

Recording data using a terminal program.
:::

> #### Calibration protocol procedure

Reviewer 2: To write both "protocol" and "procedure" is double in my view. So either or.
```suggestion
#### Calibration protocol
```

Reviewer: I think the same, too repetitive, we need to decide if protocol or procedure. I will vote for procedure since ISO also uses that word for a document which includes some steps, etc.
